spin-up-hub/src/store.js

```diff
@@ -118,15 +118,15 @@ const store = createStore({
       let data = await fetch(import.meta.env.VITE_API_HOST + "/static/hub-index-data.json")
       let documents = await data.json()
       context.state.searchIndex = lunr(function () {
-        this.field('title')
+        this.field('title', { boost: 100 })
         this.field('content')
         this.field('language', { boost: 10 })
-        this.field('author'),
-          this.field('tags', { boost: 100 })
-        this.field('keywords', { boost: 100 })
+        this.field('author', { boost: 10 }),
+        this.field('tags', { boost: 10 })
+        this.field('keywords', { boost: 10 })
```

Comment:
I think we should leave these fields as is, we want to match tags and keywords - keywords are a way to make sure we can specify alternative terms from tags. If someone wants to search for a particular author, they can search something along the lines of `author:<name>`

Reply:
Got it